core/src/test/scala/stryker4s/mutants/MutatorTest.scala

```diff
@@ -151,18 +151,4 @@ class MutatorTest extends Stryker4sSuite with TreeEquality with LogMatchers {
       "If this is not intended, please check your configuration and try again." should not be loggedAsInfo
     }
   }
-
-  describe("string interpolation") {
-
-    it("checks if the Scalameta workaround is still needed") {
```

Comment:
We can keep this test, but I am not sure about it. We did need it before to notify us if the workaround was still needed. But I think if we keep it then we are just testing Scalameta (and that already has a lot of tests)